bd_spot_wrapper/spot_wrapper/april_tag_pose_estimator.py

```diff
@@ -0,0 +1,101 @@
+from typing import List, Tuple
```

Comment:
Hi Kavit, I think this `april_tag_pose_estimator.py` could be moved to a new folder that does not directly calls spot. Happy to discuss this with you more.

Reply:
This is moved to a new folder in root of `spot-sim2real`